vk/vk/ViewController/GlobalGroupsTableViewController.swift

```diff
@@ -0,0 +1,36 @@
+// GlobalGroupsTableViewController.swift
+// Copyright © Pavel V on RoadMap. All rights reserved.
+
+import UIKit
+
+/// List of global groups searched by query
+final class GlobalGroupsTableViewController: UIGroupsTableViewController {
+    // MARK: - Private IBOutlets
+
+    @IBOutlet private var searchBar: UISearchBar!
+
+    // MARK: - Private Properties
+
+    private var filterText: String? = "" {
+        didSet {
+            loadGroups()
+        }
+    }
+
+    // MARK: - Lifecycle
+
+    override func loadGroups() {
```

Comment:
```suggestion
    func fetchGlobalGroups() {
```
Родитель ничего не должен знать ни о сервисах ни о методах загрузки